com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -515,17 +515,19 @@ internal static void NetworkBehaviourUpdate(NetworkManager networkManager)
                     s_Touched.Clear();
                     for (int i = 0; i < networkManager.ConnectedClientsList.Count; i++)
                     {
+                        s_TouchedThisClient.Clear();
                         var client = networkManager.ConnectedClientsList[i];
-                        var spawnedObjs = networkManager.SpawnManager.SpawnedObjectsList;
-                        s_Touched.UnionWith(spawnedObjs);
-                        foreach (var sobj in spawnedObjs)
+                        NetworkManager.Singleton.InterestManager.QueryFor(client, s_TouchedThisClient);
```

Comment:
I am curious why you use NetworkManager.Singleton right below the use of networkManager.ConnectedClients.
Was there a specific reason to use the singleton reference here?

Reply:
Ancient code that I forgot to notice to update...fixing